logic_mod/Blocks/ExtAnglometerBlock.cs

```diff
@@ -134,6 +134,26 @@ private float ClampAngle(float a)
             return (!(a < 0f)) ? a : (a + 360f);
         }
 
+        public Dictionary<string, object> GetAng()
+        {
+            var quaternion = Rigidbody.rotation;
+            var euler = quaternion.eulerAngles;
+            var angV = Rigidbody.angularVelocity;
+            return new Dictionary<string, object>
+            {
+                { "qw", quaternion.w },
+                { "qx", quaternion.x },
+                { "qy", quaternion.y },
+                { "qz", quaternion.z },
+                { "ex", euler.x },
+                { "ey", euler.y },
+                { "ez", euler.z},
+                { "vx", angV.x },
+                { "vy", angV.y },
+                { "vz", angV.z },

```

Comment:
This would be much better to split object, so every xyz/w will be a kind of vector:
{
    "quaternion": { "x": 1, "y": 2. "z": 3, "w": 4 },
    "euler": { "x": 1, "y": 2. "z": 3 },
    "velocity": { "x": 1, "y": 2. "z": 3  }
}
This will simplify and unify user's CPU code that will work with vectors of any kind, and also allow to add vector arifm api later.
The best practice will be adding a helper functions that convert any Vector3/Vector4 to Dictionary<string, object> and just call it 3 times here.